go.mod

```diff
@@ -2,17 +2,21 @@ module github.com/layer5io/meshery-linkerd
 
 go 1.14
 
+replace github.com/kudobuilder/kuttl => github.com/layer5io/kuttl v0.4.1-0.20200723152044-916f10574334
+
 require (
+	github.com/Azure/go-autorest/autorest/adal v0.9.0 // indirect
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751
 	github.com/ghodss/yaml v1.0.0
 	github.com/golang/protobuf v1.4.2
+	github.com/layer5io/gokit v0.1.16
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.7.0
 	golang.org/x/net v0.0.0-20200707034311-ab3426394381
 	google.golang.org/grpc v1.32.0
-	k8s.io/api v0.17.4
-	k8s.io/apiextensions-apiserver v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/client-go v0.17.4
+	k8s.io/api v0.18.8
+	k8s.io/apiextensions-apiserver v0.18.8
+	k8s.io/apimachinery v0.18.8
+	k8s.io/client-go v0.18.8
 	k8s.io/kube-aggregator v0.17.4
```

Comment:
If we already to upgrade the `k8s.io/xxx` version, we should upgrade them all at once time. If do not, may bring other issues

Reply:
@Aisuko yes you are right. I have made the changes for all of those functions. Can you review them once?

Reply:
We need fix some conflict